website/developer-docs/setup/full-dev-environment.md

```diff
@@ -5,15 +5,15 @@ title: Full development environment
 ## Requirements
 
 -   Python 3.11
--   poetry, which is used to manage dependencies, and can be installed with `pip install poetry`
+-   Poetry, which is used to manage dependencies
 -   Go 1.20
--   Node 20
+-   Node.js 20
 -   PostgreSQL (any recent version will do)
 -   Redis (any recent version will do)
 
 ## Services Setup
 
-For PostgreSQL and Redis, you can use the docker-compose file in `scripts/`.
+For PostgreSQL and Redis, you can use the `docker-compose.yml` file in `/scripts`.To use these pre-configured database instances, navigate to the `/scripts` directory in your local copy of the authentik git repo, and run `docker compose up -d`.
 You can also use a native install, if you prefer.
```

Comment:
```
You can also use a native install, if you prefer.

:::info
The PostgreSQL credentials given to authentik should have permissions for `CREATE DATABASE` and `DROP DATABASE` as authentik creates a temporary database for tests.
:::
```

We should probably add this as a note, I completely forgot about it and @kensternberg-authentik reminded me about this earlier

Reply:
Great, thanks to you both!

Reply:
```suggestion
You can also use a native install, if you prefer. 

:::info
If you use locally installed databases, the PostgreSQL credentials given to authentik should have permissions for `CREATE DATABASE` and `DROP DATABASE`, because authentik creates a temporary database for tests.
:::
```